libs/common/src/vault/services/default-cipher-encryption.service.ts

```diff
@@ -39,7 +39,7 @@ export class DefaultCipherEncryptionService implements CipherEncryptionService {
 
           return {
             cipher: Cipher.fromSdkCipher(encryptionContext.cipher)!,
-            encryptedFor: asUuid<UserId>(encryptionContext.encryptedFor),
+            encryptedFor: uuidAsString(encryptionContext.encryptedFor) as UserId,
```

Comment:
Question: I don't think we need the use of `uuidAsString` here right?
```suggestion
            encryptedFor: encryptionContext.encryptedFor as UserId,
```